code/22-decay-tree-fitter/ntuple_DTF1.py

```diff
@@ -0,0 +1,39 @@
+from GaudiConf import IOHelper
+from Configurables import DaVinci, DecayTreeTuple
+from DecayTreeTuple.Configuration import *
+
+# Stream and stripping line we want to use
+stream = 'AllStreams'
+line = 'D2hhCompleteEventPromptDst2D2RSLine'
+
+# Create an ntuple to capture D*+ decays from the StrippingLine line
+dtt = DecayTreeTuple('TupleDstToD0pi_D0ToKpi')
+dtt.Inputs = ['/Event/{0}/Phys/{1}/Particles'.format(stream, line)]
+dtt.Decay = '[D*(2010)+ -> ^(D0 -> ^K- ^pi+) ^pi+]CC'
+
+# add a kinematic fitter
+from Configurables import TupleToolDecayTreeFitter,TupleToolDecay
```

Comment:
[PEP 8](https://www.python.org/dev/peps/pep-0008/) is the style guide created and ratified by the developers who made/make Python, so it's a pretty safe bet to follow it. It's generally [not more efficient](http://stackoverflow.com/a/128577/596068) to `import` inside methods and stuff, just stick it all at the top.